experimental/framework/last-edited/src/index.ts

```diff
@@ -5,8 +5,8 @@
 
 export {
 	IFluidLastEditedTracker,
-	ILastEditDetails,
-	IProvideFluidLastEditedTracker,
+	type ILastEditDetails,
```

Comment:
I see `IFluidLastEditedTracker` is also a `const` and that must be why it's not being exported as a `type`... but I don't see that it is used as anything other than a type? Not necessarily for this PR, but the `const` seems redundant to me.